src/picongpu/include/simulationControl/MySimulation.hpp

```diff
@@ -321,6 +325,16 @@ class MySimulation : public SimulationHelper<simDim>
             this->synchrotronFunctions.init();
         }
 
+        // Initialize bremsstrahlung lookup tables, if there are species containing bremsstrahlung photons
+        if(!bmpl::empty<AllBremsstrahlungPhotonsSpecies>::value)
+        {
+            ForEach<AllBremsstrahlungPhotonsSpecies,
+                particles::bremsstrahlung::FillScaledSpectrumMap<bmpl::_1> > fillScaledSpectrumMap;
+            fillScaledSpectrumMap(forward(this->scaledBremsstrahlungSpectrumMap));
+
+            this->bremsstrahlungPhotonAngle.init();
```

Comment:
@Heikman  Is there a problem with this init method that the particle memory is not allocated at this point?

Note: memory for particles is allocated at [line 338](https://github.com/ComputationalRadiationPhysics/picongpu/pull/1504/files#diff-27ab651b1e352e1783fb08c3aed891c9R338).